ci/glci/util.py

```diff
@@ -361,14 +362,19 @@ def release_set_manifest_name(
     version: str,
     flavourset_name: str,
     build_type: glci.model.BuildType,
+    with_timestamp: bool = False,
 ):
     BT = glci.model.BuildType
 
     if build_type in (BT.SNAPSHOT, BT.DAILY):
-        return f'{gardenlinux_epoch}-{build_committish[:6]}-{flavourset_name}'
+        name = f'{gardenlinux_epoch}-{build_committish[:6]}-{flavourset_name}'
     elif build_type is BT.RELEASE:
-        return f'{version}-{flavourset_name}'
+        name = f'{version}-{flavourset_name}'
 
+    if with_timestamp:
+        name += '-' + datetime.datetime.utcnow().strftime('%Y%m%d-%H%M%S')
```

Comment:
nit: rather use `datatime.isoformat`?

Reply:
intentionally omitted to avoid the 'T'

Reply:
The string returned by isoformat can be configured, e.g.
```python3
datetime.datetime.utcnow().isoformat(' ', 'seconds')
```
results in `'2021-12-08 08:35:52'`